### ISSUE TITLE
New Issue 3

### ISSUE THREAD

**Issue by eheasley** [eheasley/TestRepo#3]
Please get to work slacker.  This is a real hard task.
- [x] Do this thing first
- [ ] Do this second
- [ ] Now everyone is happy!

**Comment by morrinene** [comment]
This sounds impossible

**Comment by eheasley** [comment]
No, it IS NOT.  I added a checklist to the original issue so you can check things off as you go.

**Comment by morrinene** [comment]
I did the first thing
